gridstatus/pjm.py

```diff
@@ -1039,36 +1040,113 @@ def get_interconnection_queue(self, verbose=False):
 
         return queue
 
-    def _get_key(self):
-        # Not using retries here, should we be?
-        settings = self._get_json(
-            "https://dataminer2.pjm.com/config/settings.json",
-            verbose=False,
+    @support_date_range(frequency=None)
+    def get_solar_forecast(self, date, end=None, verbose=False):
+        """
+        Retrieves the hourly solar forecast including behind the meter solar forecast.
+        From:  https://dataminer2.pjm.com/feed/hourly_solar_power_forecast/definition
+        Only available in past 30 days
+        """
+        if date == "latest":
+            date = "today"
+
+        df = self._get_pjm_json(
+            "hourly_solar_power_forecast",
+            start=date,
+            params={
+                "fields": "datetime_beginning_ept,datetime_beginning_utc,"
+                "datetime_ending_ept,datetime_ending_utc,evaluated_at_ept,"
+                "evaluated_at_utc,solar_forecast_btm_mwh,solar_forecast_mwh",
+            },
+            end=end,
+            filter_timestamp_name="evaluated_at",
+            interval_duration_min=60,
+            verbose=verbose,
         )
 
-        return settings["subscriptionKey"]
+        return self._parse_solar_forecast(df)
 
+    def _parse_solar_forecast(self, df):
+        df = df.rename(
+            columns={
+                "evaluated_at_utc": "Forecast Time",
```

Comment:
Let's use Publish Time. 

I know we previously were using Forecast Time, but I want to standard on "Publish Time" going forward since it more generalizable to things that might not be forecasts

Reply:
Got it, updated.